recipes/mcroni/meta.yaml

```diff
@@ -1,39 +1,46 @@
-{% set version = "1.0.1" %}
-{% set sha256 = "584afa62047ab9598b8fd89d0d39fbd265baf402103005d5efbbb71b440812a5" %} 
+{% set version = "1.0.0" %}
+{% set sha256 = "3e7c008f719a8447446d2706c8738beeb4975b1f" %}
```

Comment:
Looks like this is not a complete checksum. Please ensure it is a complete checksum